ballerina/tests/test.bal

```diff
@@ -24,320 +24,276 @@ configurable string userDN = ?;
 
 boolean isTestOnLiveServer = false;
 
-Client ldapClient = test:mock(Client);
-
-@test:BeforeSuite
-function initializeClientsForCalendarServer () returns error? {
-    if isTestOnLiveServer {
-        ldapClient = check new ({
-            hostName: hostName,
-            port: port,
-            domainName: domainName,
-            password: password
-        });
-    }
-}
+Client ldapClient = check new ({
+   hostName: hostName,
+   port: port,
+   domainName: domainName,
+   password: password
+});
 
 @test:Config {
-    enable: false
+   groups: ["add"]
 }
 public function testAddUser() returns error? {
-    UserConfig user = {
-        "objectClass": ["user", organizationalPerson, "person", "top"],
-        sn: "User",
-        cn: "User",
-        givenName: "User",
-        displayName: "User",
-        userPrincipalName: "user@ad.windows",
-        userAccountControl: "544"
-    };
-    LdapResponse val = check ldapClient->add("CN=User,OU=People,DC=ad,DC=windows", user);
-    test:assertEquals(val.resultCode, SUCCESS);
+   record {|EntryMember...;|} user = {
+       "objectClass": ["top", "person"],
+       "sn": "User",
+       "cn": "User"
+   };
+   LdapResponse val = check ldapClient->add("cn=User,dc=mycompany,dc=com", user);
+   test:assertEquals(val.resultCode, SUCCESS);
 }
 
 @test:Config {
-    enable: false,
-    dependsOn: [testAddUser]
+   dependsOn: [testAddUser]
 }
-public function testAddUserWithManager() returns error? {
-    record {} user = {
-        "objectClass": "user",
-        "sn": "New User",
-        "cn": "New User",
-        "givenName": "New User",
-        "displayName": "New User",
-        "userPrincipalName": "newuser@ad.windows",
-        "userAccountControl": "544",
-        "manager": "CN=User,OU=People,DC=ad,DC=windows"
-    };
-    LdapResponse addResult = check ldapClient->add("CN=New User,OU=People,DC=ad,DC=windows", user);
-    test:assertEquals(addResult.resultCode, SUCCESS);
+public function testAddSecondaryUser() returns error? {
+   record {|EntryMember...;|} user = {
+       "objectClass": ["top", "person"],
+       "sn": "New User",
+       "cn": "New User"
+   };
+   LdapResponse addResult = check ldapClient->add("CN=New User,dc=mycompany,dc=com", user);
+   test:assertEquals(addResult.resultCode, SUCCESS);
 }
 
 @test:Config {
-    enable: false,
-    dependsOn: [testGetUser]
+   dependsOn: [testGetUser]
 }
 public function testDeleteUserHavingManager() returns error? {
-    LdapResponse val = check ldapClient->delete("CN=New User,OU=People,DC=ad,DC=windows");
-    test:assertEquals(val.resultCode, SUCCESS);
+   LdapResponse val = check ldapClient->delete("CN=New User,dc=mycompany,dc=com");
```

Comment:
Shall we rename the variables with a contextual name ? Check for other places too.
```suggestion
   LdapResponse deleteResponse = check ldapClient->delete("CN=New User,dc=mycompany,dc=com");
```